src/apps/explorer/components/common/Dropdown/index.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { DOMAttributes, useState, useCallback, createRef, ReactElement } from 'react'
+import styled, { FlattenSimpleInterpolation } from 'styled-components'
+
+import {
+  BaseCard,
+  PositionCenterCSS,
+  PositionLeftCSS,
+  PositionRightCSS,
+  DirectionDownwardsCSS,
+  DirectionUpwardsCSS,
+  DropdownItemProps,
+  DropdownItemCSS,
+} from './styled'
+import useOnClickOutside from 'hooks/useOnClickOutside'
+
+export enum DropdownPosition {
+  center,
+  left,
+  right,
+}
+
+export enum DropdownDirection {
+  downwards = 'down',
+  upwards = 'up',
+}
+
+const Wrapper = styled.div<{ isOpen: boolean; disabled: boolean }>`
+  outline: none;
+  pointer-events: ${(props): string => (props.disabled ? 'none' : 'initial')};
+  position: relative;
+  z-index: ${(props): string => (props.isOpen ? '100' : '50')};
```

Comment:
No css expert so you can ignore my comment.

What about using "inherit" or "initial" when closed?